conans/client/api/conan_api.py

```diff
@@ -105,12 +39,13 @@ def wrapper(api, *args, **kwargs):
             sys.stdout = devnull
             sys.stderr = devnull
 
-        _init_stream(sys.stderr)
+        init_colorama(sys.stderr)
 
         try:
             log_command(f.__name__, kwargs)
-            api.create_app()
-            with environment_append(api.app.cache.config.env_vars):
+            # FIXME: Not pretty to instance here a ClientCache
+            config = ClientCache(api.cache_folder).config
```

Comment:
Yes, this will not be a problem because we will not longer inject ``config`` env-vars globally.